### PR TITLE
docs: update svn command, add ipfs hash test

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

Remove the svn checkout command from the quick start section.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [x] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: docs/quick_start.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Ensure your machine satisfies the following requirements:
 
 1. Setup the environment
 ```bash
+export OPEN_AEA_IPFS_ADDR="/dns/registry.autonolas.tech/tcp/443/https"
```
Comment (dvilelaf) [inline docs/quick_start.md:17]: See [this](https://trello.com/c/xLTJZa4z/758-set-default-remote-registry-to-be?filter=member:dvilelaf)

Comment (DavidMinarsch) [reply docs/quick_start.md:17]: Can this not be provided as part of the init command @angrybayblade ?

File: tests/test_docs/test_ipfs_hash.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+from tests.conftest import ROOT_DIR
+from tests.test_docs.helper import read_file  # type: ignore
+
+
+FETCH_COMMAND_REGEX = r"aea fetch valory\/hello_world:(?P<hash>Q.*) \-\-remote"
+HELLO_WORLD_HASH_REGEX = r"valory\/agents\/hello_world,(?P<hash>Q.*)"
+
+
+def test_hello_world() -> None:
```
Comment (DavidMinarsch) [inline tests/test_docs/test_ipfs_hash.py:33]: Nice

File: docs/quick_start.md [hunk 3]
```diff
@@ -22,22 +23,17 @@ touch Pipfile && pipenv --python 3.10 && pipenv shell
...
-```bash
-svn checkout  https://github.com/valory-xyz/open-autonomy/trunk/packages packages
-```
-
-4. Build and install your agent
+3. Get, build and install your agent
 ```bash
 aea init --reset --author default_author --ipfs
-aea fetch valory/hello_world
+aea fetch valory/hello_world:QmRXo6bdg5M91LAvFRSc19dtsuzfeh4c9Ub8VFfGrobGcg --remote
```
[the remark this answers is not in the document]
Comment (dvilelaf) [reply docs/quick_start.md:29]: Yes, and there is a problem still. I'm requested to install open-aea-cli-ipfs and even after that i get:
`
ipfshttpclient.exceptions.StatusError: HTTPError: 504 Server Error: Gateway Time-out for url: https://registry.autonolas.tech:443/api/v0/get?stream-channels=true&archive=true&compress=false&arg=QmRXo6bdg5M91LAvFRSc19dtsuzfeh4c9Ub8VFfGrobGcg`

[the remark this answers is not in the document]
Comment (dvilelaf) [reply docs/quick_start.md:29]: Working now

**Review by 8ball030 — APPROVED** [review]
LGTM

**Review by angrybayblade — APPROVED** [review]

File: docs/quick_start.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (8ball030) [reply docs/quick_start.md:29]: image

**Pull request merged**